pepper-apis/src/test/java/org/broadinstitute/ddp/route/IntegrationTestSuite.java

```diff
@@ -164,13 +164,11 @@ public static void setup() {
         LogbackConfigurationPrinter.printLoggingConfiguration();
 
         initializeDatabase();
-
         if (!isTestServerRunning()) {
             startupTestServer();
-            insertTestData();
-        } else {
-            RouteTestUtil.setupDatabasePool();
         }
+        insertTestData();
```

Comment:
Before, the parallel tests seem to fail consistently with either a NullPointerException or some kind of deadlock. This change seems to make parallel tests pass now, although I'm not sure of exactly why that it. I do suspect that before we weren't running this -> thus miss data -> thus null pointer. I'm still confused about the deadlock and didn't make any insights in that investigation.